src/main/java/uk/gov/companieshouse/api/accounts/ResourceName.java

```diff
@@ -28,6 +28,7 @@
     LOANS_TO_DIRECTORS("loans-to-directors"),
     LOANS("loans"),
     RELATED_PARTY_TRANSACTIONS("related-party-transactions"),
+    RPT_TRANSACTIONS("related-party-transactions-transactions"),
```

Comment:
I believe this one should be similar to the LOANS above, so something like:
RPT_TRANSACTIONS("transactions")